### PR TITLE
[2.5.0] Bump to release version

### PR DESCRIPTION
**Pull request by zenmonkeykstop**
## Status

Ready for review 

## Description of Changes

Towards #6554 -  bumps version to 2.5.0, updates changelog

## Testing
- [x] base is release/2.5.0
- [x] CI is passing
- [x] changelog makes sense
- [x] version updated correctly.

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -1,35 +1,11 @@
...
-* Changed installation logic to generate Python bytecode during installation (#6591)
-
-## 2.5.0~rc1
-
-### Web Applications
-
-* Added unified Redis-based session handling for the Journalist Interface and API (#6403)
-* Added supported languages list and updated i18n tooling to process all langiuages available in Weblate (#6557, #6566)
+* Added unified Redis-based session handling for the Journalist Interface and API (#6403, #6584)
+* Added supported languages list and updated i18n tooling to process all langiuages available in Weblate (#6557, #6566, #6614)
```
Comment (cfm) [inline changelog.md]:
```suggestion
* Added supported languages list and updated i18n tooling to process all languages available in Weblate (#6557, #6566, #6614)
```

**Review by cfm — CHANGES_REQUESTED** [review]
Looks good, modulo one typo (correction suggested), and then pending CI.

File: changelog.md (hunk 1 above)
Comment (zenmonkeykstop) [reply changelog.md]: ugggh, one sec

Comment (zenmonkeykstop) [reply changelog.md]: Amended version pushed with typo corrected.

**Review by cfm — APPROVED** [review]
Looks good!  Will merge once CI passes.

**Comment by cfm** [comment]
Ignoring expected failing `admin-tests`—

```
tests/test_integration.py::test_check_for_update_when_updates_not_needed FAILED
tests/test_integration.py::test_update_with_duplicate_branch_and_tag FAILED
```

—and merging.

**Pull request merged**